src/Infrastructure.EntityFramework/Repositories/Queries/CipherDetailsQuery.cs

```diff
@@ -28,7 +28,7 @@ public virtual IQueryable<CipherDetails> Run(DatabaseContext dbContext)
                         DeletedDate = c.DeletedDate,
                         Reprompt = c.Reprompt,
                         Favorite = _userId.HasValue && c.Favorites != null && c.Favorites.Contains($"\"{_userId}\":true"),
```

Comment:
Dont we have this same problem here too? I think we do these kinds of checks in some other areas too. @justindbaur `UserCipherDetailsQuery` maybe?

Reply:
yes, the same problem (Favorite is upper-case too):
https://github.com/bitwarden/server/blob/481ce38934922222c4d4054aed9c781583895723/src/Core/Services/Implementations/CipherService.cs#L645